packages/tasks/src/model-libraries.ts

```diff
@@ -280,7 +280,7 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/NVIDIA/NeMo",
 		snippets: snippets.nemo,
 		filter: true,
-		countDownloads: { wildcard: { path: "*.nemo" } },
+		countDownloads: { wildcard: { path: "*.nemo", "model_config.yaml" } },
```

Comment:
Note that counting all downloads of `model_config.yaml" would be over-estimating downloads, since I expect a number of people will check the config without actually downloading the model.

Maybe `common.pt` is a better file to use for this purpose?

Reply:
I follow the same practice as `paddlenlp` and `peft`, so I assume tracking model_config is ok?

Reply:
Yes, I think this is a non-issue. Looking at a configuration file will not trigger a download count unless downloaded.